csvkit/utilities/csvstat.py

```diff
@@ -353,7 +358,11 @@ def get_unique(table, column_id, **kwargs):
 
 
 def get_freq(table, column_id, freq_count=5, **kwargs):
-    return table.pivot(column_id).order_by('Count', reverse=True).limit(freq_count)
+    values = table.columns[column_id].values()
```

Comment:
the `.pivot` agate function is very inefficient, mainly due to `.groupby` forking a given agate table and creating a ton of tiny tables. If we just use a plain ol' python counter, we can get the same results in a much faster way.